### ISSUE TITLE
_init missing any kind of checks, redirects, etc

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lantius/flashvolunteer#6]
```
What steps will reproduce the problem?
1.  Create a new instance of flashvolunteer
2.  Go to /_init
3.  Nothing happens, that you can tell.
4.  Hit refresh.
5.  Oh shit you've doubled the DB objects!

What is the expected output? What do you see instead?

_init needs to have a flag in the database.  If the DB is not configured
then home redirects to _init for initial configuration and sets the flag. 
If the flag is set then _init does nothing.



```

Original issue reported on code.google.com by `lantatla...@gmail.com` on 30 Jan 2009 at 1:50

**Comment by GoogleCodeExporter** [comment]
```
Partially fixed in commit #31, still needs to be refactored but works for this 
milestone.
```

Original comment by `lantatla...@gmail.com` on 9 Feb 2009 at 5:35
- Added labels: **Priority-Low**
- Removed labels: **Priority-High**

**Comment by GoogleCodeExporter** [comment]
```
Fixed!
```

Original comment by `tla...@gmail.com` on 6 May 2009 at 9:46
- Changed state: **Fixed**
